packages/dashboards/addon/routes/dashboards/new.js

```diff
@@ -18,16 +18,20 @@ export default Route.extend({
    */
   user: service(),
 
+  /**
+   * @property {Service} router
+   */
+  router: service(),
+
   /**
    * Sets the model for this route
    *
    * @method model
    * @override
-   * @param title - query param containing the title of dashboard
    * @returns {DS.Model} route model
    */
-  model(pathParams, { queryParams }) {
-    queryParams = queryParams || {};
+  model() {
```

Comment:
could this be:

```js
model(params) {
```

https://octane-guides-preview.emberjs.com/release/routing/query-params/#toc_opting-into-a-full-transition